ckanext/nextgeoss/controller.py

```diff
@@ -50,8 +52,9 @@ def opensearch(self):
 
     def collections(self):
         collection_list = opensearch_config.load_settings("collections_list")
-
-        return base.render('static/collection_list.html', extra_vars={'collection_list': collection_list})
+        collection_list_newest_first = OrderedDict(reversed(collection_list.items()))
```

Comment:
I made `collection_list_newest_first` an `OrderedDict` because the original `collection_list` was also an `OrderedDict`.